spec/controllers/trips_controller_spec.rb

```diff
@@ -24,4 +24,34 @@
       end
     end
   end
+
+  describe "DELETE /trips/:id" do
+    context "When trip exsist" do
+      it "destroys the trip" do
+        trip = create(:trip)
+
+        expect do
+          delete :destroy, params: { id: trip.id }
+        end.to change(Trip, :count).by(-1)
+      end
+
+      it "sets a flash notice message and redirects to index" do
+        trip = create(:trip)
+        notice = I18n.t("flash.actions.destroy.notice", resource_name: "Trip")
+
+        delete :destroy, params: { id: trip.id }
+
+        expect(flash[:notice]).to match(notice)
+        expect(response).to redirect_to trips_path
+      end
+    end
+
+    context "When trip does not exist" do
+      it "raises record not found error" do
+        expect do
+          delete :destroy, params: { id: 3 }
+        end.to raise_error(ActiveRecord::RecordNotFound)
+      end
+    end
```

Comment:
Since we are checking for raising `ActiveRecord::RecordNotFound` elsewhere, I don't think we need to check for it at every controller action

Reply:
Done with fixup in `733f331`